Lib/fontParts/test/test_contour.py

```diff
@@ -481,29 +481,41 @@ def test_selectedBPoints_setEmptyList(self):
     # Segments
     # --------
 
-    def test_segments_offcurves_middle(self):
+    def test_segments_offcurves_end(self):
         contour, _ = self.objectGenerator("contour")
+        contour.appendPoint((84, 0), "curve")
         contour.appendPoint((0, 0), "line")
-        contour.appendPoint((117, 100), "offcurve")
-        contour.appendPoint((117, 0), "offcurve")
-        contour.appendPoint((50, 0), "curve")
+        contour.appendPoint((0, 28), "offcurve")
+        contour.appendPoint((10, 64), "offcurve")
+        contour.appendPoint((46, 64), "curve")
+        contour.appendPoint((76, 64), "offcurve")
+        contour.appendPoint((84, 28), "offcurve")
 
-        assert len(contour.segments) == 2
+        segments = contour.segments
+        assert [segment.type for segment in segments] == ["line", "curve", "curve"]
```

Comment:
Change to `self.assertEqual([segment.type for segment in segments], ["line", "curve", "curve"])`. See unit testing documentation for the assertion methods to use.

Reply:
https://docs.python.org/2/library/unittest.html#test-cases

Reply:
Oops. My mind is firmly entrenched in the pytest way.

Reply:
No worries. Pytest is really nice, but we went down the unittest road before pytest was on our radar.